docs/api/advanced/subscribe.mdx

```diff
@@ -35,6 +35,12 @@ subscribe(state.arr, () => console.log('state.arr has changed to', state.arr))
 state.arr.push('world')
 ```
 
-## Codesandbox demo in VanillaJS
+#### no usage tracking
+
```

Comment:
Basically a copy/paste of the disclaimer I added to `watch` in the last PR

Reply:
it's a bit wordy here, because no usage tracking in subscribe is by design.

Reply:
Okay, I will remove for now; I'm planning on a writing "how/why valtio works" doc for my next docs PR sometime soon (a little bit of "how is valtio different than mobx"/etc), so I can bring it back as part of that.